programs/piaware/faup1090.tcl

```diff
@@ -406,7 +406,7 @@ proc stop_faup1090_close_faup1090_socket_and_reopen {} {
 # attempt_dump1090_restart - restart dump1090 if we can figure out how to
 #
 proc attempt_dump1090_restart {{action restart}} {
-	set scripts [glob -nocomplain /etc/init.d/*dump1090*.sh]
+	set scripts [glob -nocomplain /etc/init.d/*dump1090*]
```

Comment:
The whole .sh thing is a bit of a hack. The version of dump1090 built by flightaware does include a .sh file I believe - so the "standard" install should be OK, but that's unconventional. If you're running something like dump1090-mutability, it has an init.d file that's just dump1090-mutability. To do this properly I'd follow the insserv rules:

```
       Please be aware that the following patterns of boot script file names will be not accepted by insserv:

                *.dpkg*
                *.rpm*
                *.ba*
                *.old
                *.new
                *.org
                *.orig
                *.save
                *.swp
                *.core
                *~

       with the wildcard character *.  Beside this all boot script file names beginning with one of the following characters

                $.#%_+-\*[]^:()~

       will be ignored.
```

which a tcl switch should be able to handle easily enough.